stdcommands/stdcommands.go

```diff
@@ -33,6 +33,7 @@ import (
 	"github.com/botlabs-gg/yagpdb/v2/stdcommands/memstats"
 	"github.com/botlabs-gg/yagpdb/v2/stdcommands/owldictionary"
 	"github.com/botlabs-gg/yagpdb/v2/stdcommands/ping"
+	"github.com/botlabs-gg/yagpdb/v2/stdcommands/paginatetest"
```

Comment:
remove this to along with the paginatetest command.

Reply:
Sounds good, thanks